doc/source/index.rst

```diff
@@ -39,23 +39,23 @@ Sections
 
        Documentation for the functions included in scikit-image.
 
-     - `Mission statement <values.html>`_
+   * - `Mission statement <values.html>`_
 
        Our mission, vision and values.
 
      - `Governance <skips/1-governance.html>`_
 
        How decisions are made in scikit-image.
 
-     - `Installation Steps <install.html>`_
+   * - `Installation Steps <install.html>`_
 
        How to install scikit-image.
 
-   * - `User Guide <user_guide.html>`_
+     - `User Guide <user_guide.html>`_
```

Comment:
Wait, why no asterisk here? 🤔

Reply:
@soupault there should be an asterisk every 2nd heading. Asterisk is the start of a new row.

Reply:
I only just discovered this myself, hence this element of the PR. :joy: